docs/manual/logging.rst

```diff
@@ -0,0 +1,30 @@
+=======
+Logging
+=======
+
+Sentry
+======
+
+To monitor and identify issues for OpenZaak, it is recommended to use `Sentry`_.
+If Sentry is configured for OpenZaak, all exceptions that occur within the application
+are sent to Sentry. For documentation on how to set up a project in Sentry, please refer to the official `documentation`_.
+After setting up the project, you will receive a **DSN**. The created Sentry project can be linked to OpenZaak by setting
+the environment variable ``SENTRY_DSN`` equal to this DSN.
+
+.. _`sentry`: https://sentry.io/
+.. _`documentation`: https://docs.sentry.io/error-reporting/quickstart/?platform=javascript
+
+Different logs
+==============
+
+By default, OpenZaak has multiple loggers that log to different files.
+
+- Nginx: logs produced by Nginx can be found in the ``log/nginx/`` directory.
```

Comment:
This is not correct, at least not with the Docker setup. Nginx logs are written to `stdout` of the container. You can access them via `docker logs` or `kubectl logs` depending on which platform you're running. For the vmware appliance, logs are in `/var/log/nginx/`